mxcube3/core/qutils.py

```diff
@@ -261,8 +261,12 @@ def get_queue_state():
 
     queue = queue_to_dict(include_lims_data=True)
     sample_order = queue.get("sample_order", [])
+    try:
+        current =  scutils.get_current_sample().get('sampleID', '')
```

Comment:
Do you know why get_current_sample fails ?

Reply:
I could not figure out ... :(
Happening with real hw, some mismatch during initialisation

Reply:
I guess It would be good to find out, right ?

Reply:
yes sure, but at least with this it does not fail on initialisation, solution for now?
But I will look in more detail as soon as I can run tests in the beamline

Reply:
Hm, I'm not super happy to add temporary fixes on master like this It would be good if we could get a more definitive fix.